packages/contracts/src/timelock/TimelockAsyncUnlock.sol

```diff
@@ -80,38 +90,85 @@ abstract contract TimelockAsyncUnlock is Initializable, ITimelockAsyncUnlock, Co
     /**
      * @inheritdoc ITimelockAsyncUnlock
      */
-    function requestUnlock(address owner, uint256 depositPeriod, uint256 amount)
+    function requestUnlock(address owner, uint256[] memory depositPeriods, uint256[] memory amounts)
```

Comment:
as before, please keep the "single" version of the requestUnlock and add a requestUnlockBatch.